tmt/steps/provision/testcloud.py

```diff
@@ -310,12 +310,14 @@ def _prepare_ansible(self, what):
         """ Prepare using ansible """
         # Playbook paths should be relative to the metadata tree root
         playbook = os.path.join(self.step.plan.run.tree.root, what)
+        # Prepare verbose level based on the --debug option count
+        verbose = ' -' + self.opt('debug') * 'v' if self.opt('debug') else ''
```

Comment:
Would we maybe put this line into `utils.py`? As it is repeated over and over in every plugin.

Reply:
I believe this one should be deduplicated by dynamic plugins adjustments. Will share soon.